patches/minecraft/net/minecraft/item/ItemStack.java.patch

```diff
@@ -45,7 +45,7 @@
     }
  
     private ItemStack(CompoundNBT p_i47263_1_) {
-+      super(ItemStack.class);
++      super(ItemStack.class, true);
 +      this.capNBT = p_i47263_1_.func_74764_b("ForgeCaps") ? p_i47263_1_.func_74775_l("ForgeCaps") : null;
 +      Item rawItem =
```

Comment:
It's not clear whether this is an unfinished line or you're setting it to the field below.

Reply:
Scratch that, i just can't read diffs.